README.md

```diff
@@ -356,6 +358,96 @@ reached. If no configuration file is found, Regal will use the default configura
 A custom configuration may be also be provided using the `--config-file`/`-c` option for `regal lint`, which when
 provided will be used to override the default configuration.
 
+## Ignoring Rules
+
+If some rule doesn't align with your team's preferences, don't worry! Regal is not meant to be the law, and some rules
+may not make sense for your project, or parts of it. Regal provides several different ways to ignore rules, either
+entirely, or with more granularity.
+
+### Ignoring a Rule Entirely
+
+If you want to ignore a rule entirely, set its level to `ignore` in the configuration file:
+
+```yaml
+rules:
+  style:
+    prefer-snake-case:
+      # Use camel case to comply with our naming conventions
+      level: ignore
+```
+
+**Tip**: providing a comment on ignored rules is a good way to communicate why the decision was made.
+
+### Ignoring a Rule in Some Files
+
+You can use the `ignore` attribute inside any rule configuration to provide a list of files, or patterns, that should
+be ignored for that rule:
+
+```yaml
+rules:
+  style:
+    line-length:
+      level: error
+      ignore:
+        files:
+          # ignore line length in test files to accommodate messy test data
+          - "*_test.rego"
+          # specific file used only for testing
+          - "scratch.rego"
+```
+
+### Ignoring Files Globally
+
+If you want to ignore certain files for all rules, you can use the global ignore attribute in your configuration file:
+
+```yaml
+ignore:
+  files:
+    - file1.rego
+    - "*_tmp.rego"
+```
+
+### Inline Ignore Directives
+
+If you'd like to ignore a specific violation in a file, you can add an ignore directive above the line in question, or
+alternatively on the same line to the right of the expression:
+
+```rego
+package policy
+
+import rego.v1
+
+# regal ignore:prefer-snake-case
+camelCase := "yes"
+
+list_users contains user if { # regal ignore:avoid-get-and-list-prefix
+    some user in data.db.users
+    # ...
+}
+```
+
+The format of an ignore directive is `regal ignore:<rule-name>,<rule-name>...`, where `<rule-name>` is the name of the
+rule to ignore. Multiple rules may be added to the same ignore directive, separated by commas.
+
+Note that at this point in time, Regal only considers the same line or the line following the ignore directive, i.e. it
+does not apply to entire blocks of code (like rules, functions or even packages). See [configuration](#configuration)
+if you want to ignore certain rules altogether.
+
+### Ignoring Rules via CLI Flags
+
+For development, rules, or classes of rules, may quickly be enabled or disabled using the relevant CLI flags for the
+`regal lint` command:
+
+- `--disable-all` disables **all** rules
```

Comment:
just an idea, but might make it clear why this flag exists.

```suggestion
- `--disable-all` disables **all** rules to allow specific use of `--enable`
```


I might consider moving this to the end, as it makes more sense _after_ reading about `--enable` etc... 

Or maybe they all only make sense when you know all the options... 🤔